docs/blockstates/introduction.md

```diff
@@ -26,7 +26,52 @@ The log only has one property: axis. A blockstate always has to be defined for a
 "east=false,north=false,south=false,west=false": { "model": "oak_fence_post" }
 ```
 
+!!! Note
+    The properties of a variant string of a blockstate should be dictionary ordered. For example, `"east=false,north=false,south=false,west=false"` is valid, while `"east=false,south=false,west=false,north=false"` is invalid. 
```

Comment:
I wouldn't call this "valid". This is just how the default state mapper does it.